### PR TITLE
Add in designer guidance

### LINKED ISSUE #705: "Get Started" Designer Guidance

### PR DESCRIPTION
**Pull request by taysea**
Preview: https://deploy-preview-709--keen-mayer-a86c8b.netlify.app/extend/designer-guidance
Leverages the previously posted designer copy for the new carte design and places this card under the Extend section

It'd be nice to be able to order the cards and have this one at the top.

Closes #705

### CODE REVIEW

File: aries-site/src/pages/extend/designer-guidance.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            1) Log into Figma and look for an HPE icon in your left control bar.
+            Navigate to this section, you will see all the HPE Figma teams. Look
+            for "HPE Design System" and click "Request to join".
+          </SubsectionText>
+          <SubsectionText>
+            2) Click on HPE. Here, you will see all HPE Figma teams.
+          </SubsectionText>
+          <SubsectionText>
+            3) Look for "HPE Design System" and click "Request to join".
+          </SubsectionText>
```
Comment (halocline) [inline aries-site/src/pages/extend/designer-guidance.js:80]: Semantically, this should probably be an ordered list.

File: aries-site/src/pages/extend/designer-guidance.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+          <SubsectionText>
+            To access individual files for each component, navigate to the HPE
+            Design System Figma team. If you are having trouble accessing the
+            files, you may need to{' '}
+            <Anchor
+              label="join the HPE Design System Figma team"
+              href="#joining-the-hpe-design-system-figma-team"
+            />
+            .
```
Comment (halocline) [inline aries-site/src/pages/extend/designer-guidance.js:107]:
```suggestion
            />{' '}
            first.
```

**Comment by halocline** [comment]
> It'd be nice to be able to order the cards and have this one at the top.

Agreed. Added https://github.com/hpe-design/design-system/issues/710 and put it just below Fonts (#644) in To Do for now.

File: aries-site/src/pages/extend/designer-guidance.js (hunk 1 above)
Comment (taysea) [reply aries-site/src/pages/extend/designer-guidance.js:80]: Yes, I think you're right

Comment (taysea) [reply aries-site/src/pages/extend/designer-guidance.js]: Done : )

File: aries-site/src/pages/extend/designer-guidance.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+              href="https://help.figma.com/hc/en-us/categories/360002051613-Getting-Started"
+              target="_blank"
+              rel="noopener noreferrer"
+            />{' '}
+            page is a good place to start.
+          </SubsectionText>
+          <SubsectionText>
+            Create your account with your HPE email to be able to have access to
+            all of the HPE Design System files.
+          </SubsectionText>
```
Comment (halocline) [inline aries-site/src/pages/extend/designer-guidance.js:53]:
```suggestion
          <SubsectionText>
            <Anchor
              label="Create a Figma account"
              href="https://help.figma.com/hc/en-us/articles/360039811114-Create-a-Figma-account"
              target="_blank"
              rel="noopener noreferrer"
            />{' '}
            with your HPE email address. Using your HPE email provides immediate 
            access to all HPE Design System files.
          </SubsectionText>
```

**Comment by halocline** [comment]
We probably need some sort of "What & Why Figma" --> what = like Sketch, Adobe [whatever the product is currently named] --> why = cloud based, in-browser editing and consumption, collaboration, versioning/file management, resource sharing, etc.

**Comment by taysea** [comment]
> We probably need some sort of "What & Why Figma" --> what = like Sketch, Adobe [whatever the product is currently named] --> why = cloud based, in-browser editing and consumption, collaboration, versioning/file management, resource sharing, etc.

Agreed -- I think I'll let a Figma expert elaborate on that copy

**Pull request merged**